wheel/build_wheel.sh

```diff
@@ -59,6 +59,12 @@ fi
 export PYTORCH_BUILD_VERSION=$build_version
 export PYTORCH_BUILD_NUMBER=$build_number
 
+if [[ -n "$PACKAGE_TYPE" ]]; then
+    package_type="$PACKAGE_TYPE"
+else
+    package_type = 'wheel'
+fi
+
```

Comment:
```suggestion
package_type="${PACKAGE_TYPE:-wheel}"
```

Nit, but we can use bash defaults here